samples/v1.6/Elements/Carousel.json

```diff
@@ -2,51 +2,51 @@
 	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
 	"type": "AdaptiveCard",
 	"version": "1.6",
-	"display": {
-	  "type": "carousel",
-	  "timer": 5000
-	},
-	"body": [
-	  {
-		"type": "CarouselPage",
-		"id": "firstCarouselPage",
-		"selectAction": {
-	  	"type": "Action.OpenUrl",
-	  	"title": "Click for more information about the first carousel page!",
-	  	"url": "https://adaptivecards.io/"
-		},
-		"items": [
-	  	 {
-	  		"type": "Image",
-	  		"url": "https://adaptivecards.io/content/cats/1.png",
-	  		"size": "medium"
-	  	}
-		]
-	  },
-	  {
-		"type": "CarouselPage",
-		"id": "theSecondCarouselPage",
-		"items": [
-	  	{
-	  		"type": "Image",
-	  		"url": "https://adaptivecards.io/content/cats/2.png",
-	  		"size": "medium"
-	  	}
+	"body": {
+		"type": "Carousel",
+		"timer": 5000,
+		"pages": [
+			{
+				"type": "CarouselPage",
+				"id": "firstCarouselPage",
```

Comment:
AdaptiveCards has some properties like `verticalContentAlignment`, `rtl`. Should we add it to page

Reply:
we should definitely have samples that show this off. I believe that @JeanRoca is on the hook to come up with our official samples, which we can add later...